common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -455,3 +456,199 @@ def test_teams_membership(self):
+        Given I am enrolled in a course with a team configuration and a topic
+        When I visit the Teams page for that topic
+        Then I should see the Create Team page link on bottom
+        And When I click create team link
+        Then I should see the create team page.
+        And I should see the create team header
+        And I should also see the help messages for fields.
+        """
+        self.verify_and_navigate_to_create_team_page()
+        self.assertEqual(
+            self.create_team_page.message_for_field('name'),
+            'A name that identifies your team (maximum 255 characters).'
+        )
+        self.assertEqual(
+            self.create_team_page.message_for_textarea_field('description'),
+            'A short description of the team to help other learners understand '
+            'the goals or direction of the team (maximum 300 characters).'
+        )
+        self.assertEqual(
+            self.create_team_page.message_for_field('country'),
+            'The country that team members primarily identify with.'
+        )
+        self.assertEqual(
+            self.create_team_page.message_for_field('language'),
+            'The language that team members primarily use to communicate with each other.'
+        )
+
+    def test_user_can_see_error_message_for_missing_data(self):
+        """
+        Scenario: The user should be able to see error message in case of missing required field.
+        Given I am enrolled in a course with a team configuration and a topic
+        When I visit the Create Team page for that topic
+        Then I should see the Create Team header and form
+        And When I click create team button without filling required fields
+        Then I should see the error message and highlighted fields.
+        """
+        self.verify_and_navigate_to_create_team_page()
+        self.create_team_page.submit_form()
+
+        self.assertEqual(
+            self.create_team_page.validation_message_text,
+            'Check the highlighted fields below and try again.'
+        )
+        self.assertTrue(self.create_team_page.error_for_field(field_id='name'))
+        self.assertTrue(self.create_team_page.error_for_field(field_id='description'))
+
+    def test_user_can_see_error_message_for_incorrect_data(self):
+        """
+        Scenario: The user should be able to see error message in case of increasing length for required fields.
+        Given I am enrolled in a course with a team configuration and a topic
+        When I visit the Create Team page for that topic
+        Then I should see the Create Team header and form
+        When I add text > than 255 characters for name field
+        And I click Create button
+        Then I should see the error message for exceeding length.
+        """
+        self.verify_and_navigate_to_create_team_page()
+        self.create_team_page.submit_form()
```

Comment:
Is this line a mistake? It doesn't make sense in the context of what your test is doing:
* go to create team page
* submit unfilled form
* (error message is rendered at this point)
* fill out form, don't submit
* expect error message which has appeared before you filled in any data

Could you please re-write this test to call submit form *after* you've filled in >255 characters?